nextjs-nabil/src/context.js

```diff
@@ -8,23 +8,27 @@ import {
 import { useRouter } from "next/router";
 import { useMediaQuery } from "react-responsive";
 
-export const LoginContext = createContext();
+export const AppContext = createContext();
 
-const LoginProvider = (props) => {
+const AppProvider = (props) => {
   const { children } = props;
   const [isAuthenticated, setisAuthenticated] = useState("false");
   const [alertFailLogin, setalertFailLogin] = useState(false);
+  const [dataTeams, setDataTeams] = useState([]);
+  const [dataGames, setDataGames] = useState([]);
+  const [dataPlayers, setDataPlayers] = useState([]);
+  const [moreData, setmoreData] = useState([]);
+  const [valueBottomNavbar, setvalueBottomNavbar] = useState(0);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isLoadingMore, setIsLoadingMore] = useState(false);
+
   const router = useRouter();
-  // const isDesktopOrLaptop = useMediaQuery({
-  //   query: "(min-width: 1224px)",
-  // });
-  // const isBigScreen = useMediaQuery({ query: "(min-width: 1824px)" });
-  // const isTabletOrMobile = useMediaQuery({ query: "(max-width: 1224px)" });
-  // const isPortrait = useMediaQuery({ query: "(orientation: portrait)" });
-  // const isRetina = useMediaQuery({ query: "(min-resolution: 2dppx)" });
```

Comment:
with this now removed, how are you handling your responsive layout?

Reply:
I used the css directly, mostly applying it directly in the mui component using "sx"